src/legacy/core_plugins/kibana/public/local_application_service/_local_application_service.scss

```diff
@@ -1,5 +0,0 @@
-.kbnLocalApplicationWrapper {
```

Comment:
Does this wrapper no longer exist and if so are we sure that all apps render correctly without it?

Reply:
Yes, the "local application service" along with related styles is going away completely.

Reply:
And all apps are rendering correctly without this particular element and the flex styles?

Reply:
In current master it's not used at all - all apps migrated away already (which includes separate styles per plugin)